functions/functionsv2/helloauditlog/hello_auditlog.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_log_cloudevent]
+
+// Package helloworld provides a set of Cloud Functions samples.
+package helloworld
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/cloudevents/sdk-go/v2/event"
+	auditevents "github.com/googleapis/google-cloudevents-go/cloud/audit/v1"
+)
+
+// HelloAuditLog receives a Cloud Audit Log event, and logs a few fields.
+func HelloAuditLog(ctx context.Context, e event.Event) error {
+	log.Printf("Event Type: %s", e.Type())
+	log.Printf("Subject: %s", e.Subject())
```

Comment:
non-blocking: canonical sample includes "Method" property in output, and it looks like output order is different. Perfect alignment is not required, but I think it's useful to help polyglot developers contrast the code, and sometimes reveals unintended, subtle variations in the developer experience.

Reply:
This may warrant a followup issue for the cloudevent library, because the method name property is not present in the [event.Event object](https://pkg.go.dev/github.com/cloudevents/sdk-go/v2@v2.6.1/event#Event).
How do you suggest we proceed?

Reply:
The inconsistency i mentioned here has now been corrected in the canonical sample. this is now consistent.